modules/codemod/lib/v5/spec/renameSpacingExpressionsAndTypeReferences.spec.ts

```diff
@@ -0,0 +1,211 @@
+import {expectTransformFactory} from './expectTransformFactory';
+import transformer from '../renameSpacingExpressionsAndTypeReferences';
+const context = describe;
+
+const expectTransform = expectTransformFactory(transformer);
+
+describe('Canvas Kit Spacing Member Expressions and Type References Codemod', () => {
+  context('when used in CSS object styles', () => {
+    it('should ignore files without canvas-kit imports', () => {
+      const input = `
+      import { spacing } from "@workday/canvas-space-web";
+      
+      const styles = {
+        padding: spacing.medium,
+        margin: spacing.small,
+      };`;
+      const expected = `
+      import { spacing } from "@workday/canvas-space-web";
+      
+      const styles = {
+        padding: spacing.medium,
+        margin: spacing.small,
+      };`;
+
+      expectTransform(input, expected);
+    });
+
+    it('should properly transform spacing to space', () => {
+      const input = `
+      import { space } from "@workday/canvas-kit-react/core";
```

Comment:
This doesn't quite look right to me, but I guess this is because both `spacing` member expressions and import remapping aren't run in this test and the `import` is needed for the filtering. Still looks weird since `spacing` looks like it is coming out of nowhere.

Reply:
Yeah, I agree it's odd. It's assuming the order of the v5 codemod, but that doesn't really matter here. I'll update it.

Reply:
I'm going to combine the two transformers to resolve this.